playwright.config.ts

```diff
@@ -0,0 +1,11 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
```

Comment:
This seems to be the default template, but as there are no real e2e tests - hey it is a static page  - I suggest to remove it :-)